app/schemas/serviceMetric.py

```diff
@@ -39,8 +40,14 @@ class Config:
 
 # Properties to return to client
 class ServiceMetric(ServiceMetricInDBBase):
-    pass
-
+    metricId: int
+    metric_name: str
+    value: Union[float, int, str, bool]
+    date: datetime
+    weight: Optional[int]
```

Comment:
weight is not optional! you can't link a metric with service without using a scorecard and it's required to set a weight for the metric when linked with a scorecard.